agent_store/src/postgres.rs

```diff
@@ -67,8 +67,8 @@ where
     }
 }
 
-pub async fn issuance_state(event_publishers: Vec<Box<dyn EventPublisher>>) -> IssuanceState {
-    let pool = default_postgress_pool(&config!("db_connection_string", String).unwrap()).await;
+pub async fn issuance_state(event_publishers: Vec<Box<dyn EventPublisher>>, connection_string: &str) -> IssuanceState {
```

Comment:
Why do we add the connection_string to the function's signature here? Can't we just directly retrieve it from the `config()`?

Reply:
Refactored ✅